src/App.jsx

```diff
@@ -2,43 +1,26 @@
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
+import { Tabs } from './components/Tabs/Tabs';
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">
-              Tab 1
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">
-              Tab 2
-            </a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">
-              Tab 3
-            </a>
-          </li>
-        </ul>
-      </div>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
+  const [onTabSelected, setOnTabSelected] = useState(tabs[0]);
+  return (<div className="section">
+    <h1 className="title">{`Selected tab is ${onTabSelected.title}`}</h1>
 
-      <div className="block" data-cy="TabContent">
-        Some text 1
-      </div>
-    </div>
-  </div>
-);
+    <Tabs 
+    tabs={tabs} 
+    activeTabId={activeTabId} 
+    onTabSelected={onTabSelected} 
```

Comment:
The `onTabSelected` prop should be a callback function, not the selected tab object. You should pass a function that handles the tab selection logic, which will be called when a non-active tab is selected.